### PR TITLE
Update upgrade-to-Teams-execute-SkypeforBusinessOnline.md

### PR DESCRIPTION
**Pull request by AlexanderHolmeset**
org wide settigns menu does not excist anymore  and some other name changes

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/upgrade-to-Teams-execute-SkypeforBusinessOnline.md [hunk 1]
```diff
@@ -53,12 +53,12 @@ Follow these steps to upgrade all of your users to Teams at one time.
...
 ### Step 1: Notify the users of the change (optional)
 
-1. In the Microsoft Teams admin center, select **Org-wide settings** > **Teams upgrade**.
+1. In the Microsoft Teams admin center, select **Teams** > **Teams upgrade settings**.
 2. Under **Coexistence mode**, change the **Notify Skype for Business users that an upgrade to Teams is available** switch to **On**.
 
 ### Step 2: Set the coexistence mode to TeamsOnly for the organization
 
-1. In the Microsoft Teams admin center, select **Org-wide settings**.
+1. In the Microsoft Teams admin center, select **Teams** > **Teams upgrade settings**..
```
Comment (JohanFreelancer9) [inline Teams/upgrade-to-Teams-execute-SkypeforBusinessOnline.md:61]:
```suggestion
1. In the Microsoft Teams admin center, select **Teams** > **Teams upgrade settings**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @AlexanderHolmeset, kindly commit the suggested change to correct the minor typo. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**